commands/daemon.go

```diff
@@ -255,16 +254,13 @@ Note that jobs are not persisted between restarts of the daemon. See
 			node.Override(new(*schedule.Scheduler), schedule.NewSchedulerDaemon),
 			node.Override(new(*storage.Catalog), modules.NewStorageCatalog),
 			node.Override(new(*distributed.Catalog), modules.NewQueueCatalog),
-			node.Override(new(*lutil.CacheConfig), modules.CacheConfig(cacheFlags.BlockstoreCacheSize, cacheFlags.StatestoreCacheSize)),
```

Comment:
Don't remove these. They are required.

Reply:
But there is no `CachingBlockstore` in lotus anymore. How to fix it?